energy.md

```diff
@@ -45,10 +45,15 @@ GCAM models depletable resources (oil, unconventional oil, natural gas, coal, an
 <img src="gcam-figs/gas_supply_curve.png" width="750" height="300" />
 {: .fig}
 <img src="gcam-figs/coal_supply_curve.png" width="750" height="300" />
-**Illustrative examples of supply curves and totall supplies for fossil resources in GCAM (from GCAM 5.1.2)**
+**Illustrative examples of supply curves and totall supplies for fossil resources in GCAM (from GCAM 5.3)**
```

Comment:
Let's fix the typo: `totall`

Also let's drop the (from GCAM XX). The documentation will be archived for previous versions so the relationship should be implied.

Reply:
Fixed typo. Also removed version number here.